wiki/Manual.md

```diff
@@ -7297,6 +7297,29 @@ Possible reasons for this exception:
   - `InvalidOrder`: the base class for all exceptions related to the unified order API.
   - `OrderNotFound`: when you are trying to fetch or cancel a non-existent order.
 
+### Notes about expired requests
+
+Sometimes, users might get an error like:
+> "Timestamp for this request is outside of the recvWindow."
+> "invalid request, please check your server timestamp or recv_window param"
+> "Timestamp for this request was 1000ms ahead of the server's time" ...
+
+Such issues might be caused by several reasons:
+- Your device's clock is not synchronized correctly to the global datetime, thus the discrepancy arises. Try sync your device time correctly so your OS clock is accurate by milliseconds. However, this should not be a one-time action, you should set-up your system to synchronize periodically (eg. once in a hour) if you want to avoid such issues.
+- If your device timestamp is correctly synchronized, but there is an internet-day and your request takes more than X seconds (might be permanent number, eg. `5` or might be configurable, depends on specific exchange), in such case exchange might invalidate your request.
+- If you still get that issue, try to check the difference between your & exchange's engine timestamps like:
+```
+for i in range(0, 20):
+    local_time = exchange.milliseconds()
+    exchange_time = await exchange.fetch_time()
+    print(exchange_time - local_time)
+```
+- You might also try to set:
+A) `exchange.options['adjustForTimeDifference'] = True`
+or
+B) `exchange.options['recvWindow'] = 10000` (eg. increase to 10 seconds)
```

Comment:
the thing is, only 15 out of 145 support recvWindow

<img width="404" height="425" alt="image" src="https://github.com/user-attachments/assets/6684b6a9-a27c-4f8f-a2c2-8d89b33cf43c" />